boilerplate/app/i18n/ar.ts

```diff
@@ -1,122 +1,8 @@
-    contributeToIgniteLink: "ساهم في Ignite",
-    theLatestInReactNativeTitle: "الاحدث في React Native",
-    theLatestInReactNative: "نخن هنا لنبقيك محدثاً على جميع React Native التي تعرضها",
-    reactNativeRadioLink: "راديو React Native",
-    reactNativeNewsletterLink: "نشرة اخبار React Native",
-    reactNativeLiveLink: "مباشر React Native",
-    chainReactConferenceLink: "مؤتمر Chain React",
-    hireUsTitle: "قم بتوظيف Infinite Red لمشروعك القادم",
-    hireUs:
-      "سواء كان الامر يتعلّق بتشغيل مشروع كامل او اعداد الفرق بسرعة من خلال التدريب العلمي لدينا، يمكن ان يساعد Infinite Red اللامتناهي في اي مشروع محلي يتفاعل معه.",
-    hireUsLink: "ارسل لنا رسالة",
-  },
-  demoShowroomScreen: {
-    jumpStart: "مكونات او عناصر لبدء مشروعك",
-    lorem2Sentences:
-      "عامل الناس بأخلاقك لا بأخلاقهم. عامل الناس بأخلاقك لا بأخلاقهم. عامل الناس بأخلاقك لا بأخلاقهم",
-    demoHeaderTxExample: "ياي",
-    demoViaTxProp: "عبر `tx` Prop",
-    demoViaSpecifiedTxProp: "Prop `{{prop}}Tx` عبر",
-  },
-  demoDebugScreen: {
-    howTo: "كيف",
-    title: "التصحيح",
-    tagLine: "مبروك، لديك نموذج اصلي متقدم للغاية للتفاعل هنا. الاستفادة من هذه النمذجة",
-    reactotron: "Reactotron ارسل إلى",
-    reportBugs: "الابلاغ عن اخطاء",
-    demoList: "قائمة تجريبية",
-    demoPodcastList: "قائمة البودكاست التجريبي",
-    androidReactotronHint:
-      "اذا لم ينجح ذللك، فتأكد من تشغيل تطبيق الحاسوب الخاص Reactotron، وقم بتشغيل عكس adb tcp:9090 \ntcp:9090 من جهازك الطرفي ، واعد تحميل التطبيق",
-    iosReactotronHint:
-      "اذا لم ينجح ذلك، فتأكد من تشغيل تطبيق الحاسوب الخاص ب Reactotron وأعد تحميل التطبيق",
-    macosReactotronHint: "اذا لم ينجح ذلك، فتأكد من تشغيل الحاسوب ب Reactotron وأعد تحميل التطبيق",
-    webReactotronHint: "اذا لم ينجح ذلك، فتأكد من تشغيل الحاسوب ب Reactotron وأعد تحميل التطبيق",
-    windowsReactotronHint:
-      "اذا لم ينجح ذلك، فتأكد من تشغيل الحاسوب ب Reactotron وأعد تحميل التطبيق",
-  },
-  demoPodcastListScreen: {
-    title: "حلقات إذاعية React Native",
-    onlyFavorites: "المفضلة فقط",
-    favoriteButton: "المفضل",
-    unfavoriteButton: "غير مفضل",
-    accessibility: {
-      cardHint: "انقر مرّتين للاستماع على الحلقة. انقر مرّتين وانتظر لتفعيل {{action}} هذه الحلقة.",
-      switch: "قم بالتبديل لاظهار المفضّلة فقط.",
-      favoriteAction: "تبديل المفضلة",
-      favoriteIcon: "الحلقة الغير مفضّلة",
-      unfavoriteIcon: "الحلقة المفضّلة",
-      publishLabel: "نشرت {{date}}",
-      durationLabel: "المدّة: {{hours}} ساعات {{minutes}} دقائق {{seconds}} ثواني",
-    },
-    noFavoritesEmptyState: {
-      heading: "هذا يبدو فارغاً بعض الشيء.",
-      content:
-        "لم تتم اضافة اي مفضلات حتى الان. اضغط على القلب في إحدى الحلقات لإضافته الى المفضلة.",
-    },
-  },
+  ...demo.ar,
   // @demo remove-block-end
```

Comment:
These `remove-block-end` comments were specific to the `remove-block-start` ones.

This is for when you `npx ignite-cli new MyApp --remove-demo` the marked areas would have been cleaned up.

🤔 Our test should probably be improved that no `@demo` remains in any file after `--remove-demo` has run. I'm thinking this might need some adjustments. With this flag, I think `welcomeScreen`, `errorScreen`, and `emptyStateComponent` are the ones that need to say (perhaps `common`)